test/verifiedTests/reconstruction/testModelManipulation/testUpdateGenes.m

```diff
@@ -0,0 +1,41 @@
+% The COBRAToolbox: testUpdateGenes.m
+%
+% Purpose:
+%     - testUpdateGenes tests updateGenes
+%
+% Authors:
+%     - Uri David Akavia August 2017
+
+% save the current path
+currentDir = pwd;
+
+fileDir = fileparts(which('testUpdateGenes'));
+cd(fileDir);
+
+modelDirectory = strcat('..', filesep, '..', filesep, '..', filesep, 'models', filesep);
+
+model = readCbModel(strcat(modelDirectory, 'Recon2.v04.mat'));
```

Comment:
May you use here:
```
model = readCbModel([CBTDIR filesep 'test' filesep 'models' filesep 'Recon2.v04.mat']);
```
There is no need for a relative `modelDirectory`, which is not very robust.